src/homepageExperience/components/steps/cli/InitializeClient.tsx

```diff
@@ -123,10 +145,31 @@ export const InitializeClient: FC<OwnProps> = ({
         recommend using a token with more specific permissions in the long-term.
         You can edit your tokens anytime from the app.
       </p>
-      <h2 style={{marginTop: '48px', marginBottom: '8px'}}>Create a bucket</h2>
```

Comment:
Not a ton of styling here, but maybe we should export these to a class. Also see this [article](https://www.aaronbuxbaum.com/how-object-literals-hurt-performance/) Bucky linked recently re props being assigned an object literal.